addons/filesystem/functions/fnc_device_addFile.sqf

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
Same syncing issues as above. Make function only executable on server or handle filesystem syncing with more care.

Reply:
You are right. I will restrict this function to only run on server.